library/asset/sprite/src/asset-sprite.cpp

```
@@ -0,0 +1,638 @@
+        auto & bone = info.data.skeleton.bones[it];
+        auto & libraryBone = library.data.skeleton.bones[it];
+        libraryBone = {
+          .name = std::string(bone.name.contents, bone.name.len),
+          .parentIndex = bone.parentIndex,
+          .originOffset = bone.originOffset,
+          .animations = {},
+        };
+        libraryBone.animations.animations.resize(
+          bone.animationLibrary.animationCount
+        );
+        memcpy(
```

cppcheck (2.10-2)

[error] Using 'memcpy' on struct that contains a 'std

(memsetClass)